### PR TITLE
[OASIS-7774] Add accept test for v1/datafile endpoint

### PR DESCRIPTION
**Pull request by Mat001**
## Summary
- Acceptance tests for /v1/datafile were missing from the beginning, adding it here
- also fixing failing tests because of pytest-clarity library was updated and now uses different options that previously used

The "why", or other context.

## Issues
- https://optimizely.atlassian.net/browse/OASIS-7774
- https://optimizely.atlassian.net/browse/OASIS-7793

### CODE REVIEW

File: tests/acceptance/test_acceptance/test_datafile.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                                       "variables": []}]}],
+                               "id": "16943340293"}],
+                       "typedAudiences": [], "variables": [], "version": "4"}
+
+
+def test_datafile_200(session_obj):
+    """
+    Normally a good practice is to haev expected response as a string likein other tests.
+    Here we are exceptionally making expected response a dict for easier comparison.
+    String was causing some issues extra with spaces.
```
Comment (keppel2) [inline tests/acceptance/test_acceptance/test_datafile.py:157]: This is fine and should make it less brittle.

File: tests/acceptance/test_acceptance/test_datafile.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+                                       "id": "16925940659",
+                                       "key": "16925940659",
+                                       "variables": []}]}],
+                               "id": "16943340293"}],
+                       "typedAudiences": [], "variables": [], "version": "4"}
+
+
+def test_datafile_200(session_obj):
+    """
+    Normally a good practice is to haev expected response as a string likein other tests.
```
Comment (keppel2) [inline tests/acceptance/test_acceptance/test_datafile.py:155]: Some typos, but no need to fix.

**Review by keppel2 — APPROVED** [review]

File: tests/acceptance/test_acceptance/test_datafile.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
+from tests.acceptance.helpers import ENDPOINT_DATAFILE
+from tests.acceptance.helpers import create_and_validate_request_and_response
```
Comment (msohailhussain) [inline tests/acceptance/test_acceptance/test_datafile.py:2]: write both in single line.

File: tests/acceptance/test_acceptance/test_datafile.py [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+
+expected_response = {"accountId": "10845721364", "anonymizeIP": True,
```
Comment (msohailhussain) [inline tests/acceptance/test_acceptance/test_datafile.py:5]: if possible can you place in a separate file

File: tests/acceptance/test_acceptance/test_datafile.py [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+                                   {
+                                       "featureEnabled": True,
+                                       "id": "16925940659",
+                                       "key": "16925940659",
+                                       "variables": []}]}],
+                               "id": "16943340293"}],
+                       "typedAudiences": [], "variables": [], "version": "4"}
+
+
+def test_datafile_200(session_obj):
```
Comment (msohailhussain) [inline tests/acceptance/test_acceptance/test_datafile.py:153]: instead of 200, test_datafile_success more suitable.

**Review by msohailhussain — CHANGES_REQUESTED** [review]
please address.

File: tests/acceptance/test_acceptance/test_datafile.py (hunk 4 above)
Comment (Mat001) [reply tests/acceptance/test_acceptance/test_datafile.py:5]: yeah, was thinking about that

**Review by msohailhussain — APPROVED** [review]
lgtm

**Pull request merged**